docs/pelion_ready/device-management-config.md

```diff
@@ -2,47 +2,47 @@
 
 Device Management configuration has five distinct areas:
 
-- Connectivity - the transport type for the device to connect to the device management service.
+- Connectivity - the transport type for the device to connect to the Device Management service.
```

Comment:
: instead of - ?

Reply:
Already one : above, so I decided to leave this as is.